airbyte_cdk/sources/concurrent_source/concurrent_source.py

```diff
@@ -44,6 +44,7 @@ def create(
         slice_logger: SliceLogger,
         message_repository: MessageRepository,
         timeout_seconds: int = DEFAULT_TIMEOUT_SECONDS,
+        queue: Optional[Queue[QueueItem]] = None
```

Comment:
Since the MessageRepository also needs to have access to the queue, we need to have the queue passed here instead of being created